### PR TITLE
[Merged by Bors] - chore: reduce use of Init.Data.Int.CompLemmas

### PR DESCRIPTION
**Pull request by kim-em**
`Mathlib.Init.Data.Int.CompLemmas` was ported from core, and was never really intended for outside use. Nevertheless, people started using it. (Surprise!)

This PR removes one out of the two uses in Mathlib.

If anyone would like to do the other in a separate PR, please do! You would need to reprove
```
theorem natAbs_add_nonneg {a b : ℤ} (wa : 0 ≤ a) (wb : 0 ≤ b) :
    Int.natAbs (a + b) = Int.natAbs a + Int.natAbs b := sorry

theorem natAbs_add_neg {a b : ℤ} (wa : a < 0) (wb : b < 0) :
    Int.natAbs (a + b) = Int.natAbs a + Int.natAbs b := sorry
```

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Int/Basic.lean [hunk 1]
```diff
@@ -7,6 +7,7 @@ import Mathlib.Init.Data.Int.Order
 import Mathlib.Data.Int.Cast.Basic
 import Mathlib.Algebra.Ring.Basic
 import Mathlib.Order.Monotone.Basic
+import Mathlib.Tactic.Lift
```
Comment (digama0) [inline Mathlib/Data/Int/Basic.lean:10]: why is this here?

Comment (semorrison) [inline Mathlib/Data/Int/Basic.lean:10]:
```suggestion
```

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge
